src/components/Product/index.tsx

```diff
@@ -1,32 +1,54 @@
 import styled from "@emotion/styled";
 import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
 
 const Product = () => {
+  const [isLiked, setIsLiked] = useState(false);
+
+  const handleLike = (e: any) => {
+    e.preventDefault();
```

Comment:
h1: any라는 타입은 최대한 지양해주세요!
타입스크립트의 장점이 최대한 구체적으로 타입을 지정함으로써 자바스크립트의 동적인 타입 지정을 보완한 것인데, any를 사용하면 그 장점이 줄어듭니다. 최대한 구체적으로 명시 해주는 것이 좋습니다.  또한, any와 비슷한 타입으로 unknown이 있는데 이 둘의 차이점은 아래를 참조해주시면 좋을 거 같습니다. 

참고자료: https://developer-talk.tistory.com/198#:~:text=unknown%20%ED%83%80%EC%9E%85%EC%9D%80%20%EB%8B%A8%EC%96%B4%EC%9D%98,%EC%97%B0%EC%82%B0%EC%9D%84%20%ED%95%A0%20%EC%88%98%20%EC%97%86%EC%8A%B5%EB%8B%88%EB%8B%A4.
타입참고자료: https://www.codegrepper.com/code-examples/typescript/event.preventDefault%28%29%3B+typescript+type
```suggestion 
// input이라고 하면,
  const handleLike = (event: ChangeEvent<HTMLInputElement>) => {
    event.preventDefault();
```